internal/engine/interpreter/interpreter.go

```diff
@@ -261,6 +264,16 @@ func (si *stackIterator) FunctionDefinition() api.FunctionDefinition {
 	return si.fn.definition()
 }
 
+// SourceOffset implements experimental.StackIterator.
```

Comment:
nit: to avoid introducing style 3 or 4 of this boilerplate ;)

```suggestion
// SourceOffset implements the same method as documented on experimental.StackIterator.
```